detection_rules/config.py

```diff
@@ -100,7 +101,7 @@ def get_test_names(self, formatted: bool = False) -> (List[str], List[str]):
 
         combined_tests = sorted(set(defined_tests + self.tests_by_patterns(*patterns)))
 
-        if self.unit_tests.test_only is not None:
```

Comment:
revert this and update `UnitTest.__post_init__` assertion error to

```python
        assert (self.bypass is not None and self.test_only is not None), 'Cannot set both `test_only` and `bypass` in test_config!'
```

Reply:
Strange duplicate.